### PR TITLE
Update blending script to send email when UK data is missing

### PR DESCRIPTION
**Pull request by YaliMao-NOAA**
Update blending script to send email when UK data is missing
    1. usonly.emailbody is differentiated for each forecast hour with missing UK data
    2. Remove the condition of sending UK unblended data if US unblended data is missing. It won't happen because the job itself won't get triggered if US unblended data is missing
    3. Update email list
    
Add an ecflow client test script

### CODE REVIEW

File: dev/ecf/run.nrt_ecflow.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ecflow_client --restart
+
+cd $DIR_ROOT/ecf/def
+
+# echo "yes" | ecflow_client --delete=/$suitename
+# ecflow_client --load $PWD/$suitename.def
+
+# Replace: it can either load a new suite def or
+# replace with a new suite def UNLESS a job is active for the current suite
+ecflow_client --replace=/$suitename $PWD/$suitename.def
```
Comment (aerorahul) [inline dev/ecf/run.nrt_ecflow.sh]: If the suite is replaced everytime, its not a NRT run.

File: dev/ecf/run.nrt_ecflow.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+if [[ $(hostname) =~ ^[d][login|dxfer] ]]  ; then
+    export ECF_HOST="ddecflow01"
+elif [[ $(hostname) =~ ^[c][login|dxfer] ]]  ; then
+    export ECF_HOST="cdecflow01"
+fi
+echo $ECF_HOST
+module load ecflow
+
+# Make sure ecflow_server is not halted
+ecflow_client --restart
```
Comment (aerorahul) [inline dev/ecf/run.nrt_ecflow.sh]: If you have to restart the server with every execution of the script, there is a problem in the setup.

File: dev/ecf/run.nrt_ecflow.sh [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+cd $DIR_ROOT/ecf/def
+
+# echo "yes" | ecflow_client --delete=/$suitename
+# ecflow_client --load $PWD/$suitename.def
+
+# Replace: it can either load a new suite def or
+# replace with a new suite def UNLESS a job is active for the current suite
+ecflow_client --replace=/$suitename $PWD/$suitename.def
+
+ecflow_client --begin $suitename
```
Comment (aerorahul) [inline dev/ecf/run.nrt_ecflow.sh]: This will obliterate an existing NRT run

File: dev/ecf/run.nrt_ecflow.sh [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+./setup_ecf.sh -x x001
+
+suitename="wafsx001"
+
+# set ECF_HOST according to which WCOSS2 machine
+if [[ $(hostname) =~ ^[d][login|dxfer] ]]  ; then
+    export ECF_HOST="ddecflow01"
+elif [[ $(hostname) =~ ^[c][login|dxfer] ]]  ; then
+    export ECF_HOST="cdecflow01"
+fi
```
Comment (aerorahul) [inline dev/ecf/run.nrt_ecflow.sh]: you could just put this in your `.bashrc`

File: dev/ecf/run.nrt_ecflow.sh [hunk 5]
```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```
Comment (aerorahul) [inline dev/ecf/run.nrt_ecflow.sh]: The commands in this script should be in a README.  Running this script is like setting up a NRT run every time.
Instead, use `ecflow_client` calls once the suite has been created by `setup_ecf.sh`.

**Review by aerorahul — COMMENTED** [review]
Updates to scripts/ and Release notes looks ok. But look at the comments for `run.nrt_ecflow.sh`. In my opinion, it is more of a documentation on how to run in NRT, than a script.

File: dev/ecf/run.nrt_ecflow.sh (hunk 2 above)
Comment (YaliMao-NOAA) [reply dev/ecf/run.nrt_ecflow.sh]: I don't need to restart the server. It seems ok 'ecflow_client --restart' even though the server is running.

File: dev/ecf/run.nrt_ecflow.sh (hunk 1 above)
Comment (YaliMao-NOAA) [reply dev/ecf/run.nrt_ecflow.sh]: It's one time loading. It works as 'load'.
You are right, I should use this script for PDY related ecflow.

**Review by aerorahul — APPROVED** [review]

**Pull request merged**